mcs/class/Mono.Debugger.Soft/Mono.Debugger.Soft/FieldInfoMirror.cs

```diff
@@ -162,6 +162,21 @@ public Boolean IsNotSerialized
 			}
 		}
 
+		public int IsFixedSize
+		{
```

Comment:
If its not a boolean, it shouldn't be named IsSomething

Reply:
Is this new public property needed ?

Reply:
Yes, it's not a boolean, it's the size, but I need to access it to decide on WriteValue if it's a fixed size array.

Reply:
The property doesn't need to be public. Also, the return value needs to be cached, since this call might make multiple roundtrips to the debuggee. Also, it should be possible to just return false if the type of the field is not a valuetype.